src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -9200,7 +9231,41 @@ int main(int argc, const char **argv)
       return ret;
     }
 
-    if (opt_cmd == OPT::DEDUP_ABORT || opt_cmd == OPT::DEDUP_PAUSE || opt_cmd == OPT::DEDUP_RESUME) {
+    if (opt_cmd == OPT::DEDUP_THROTTLE) {
+      bufferlist urgent_msg_bl;
+      urgent_msg_t urgent_msg = URGENT_MSG_THROTTLE;
```

Comment:
`urgent_msg_t`  and `throttle_message_t` are dedup specific types, right?
can we add a "dedup" namespace here.

Reply:
They are under rgw::dedup name space.
We are using namespace rgw::dedup; (see line 9184/9215 at the beginning of the OPT:DEDUP* block)